scripts/build.sh

```diff
@@ -14,12 +14,10 @@ VERSION=$($(dirname $0)/pkg-version --full)
 LDFLAGS="-X github.com/gluster/glusterd2/gdctx.GlusterdVersion=$VERSION"
 BIN=$(basename $(go list -f '{{.ImportPath}}'))
 
-## This so that mgmt builds without the libvirt and augeas resrouces on centos-ci
-## TODO Remove this once mgmt can be built engine only without resources
-export GOTAGS='novirt noaugeas'
-
 echo "Building $BIN $VERSION"
-go build -ldflags "${LDFLAGS}" -o $OUTDIR/$BIN || exit 1
+## Adding GOTAGS so that mgmt builds without the libvirt and augeas resrouces on centos-ci
+## TODO Remove this once mgmt can be built engine only without resources
+GOTAGS='novirt noaugeas' go build -ldflags "${LDFLAGS}" -o $OUTDIR/$BIN || exit 1
```

Comment:
Nope. use:
`go build -ldflags "${LDFLAGS}" -o $OUTDIR/$BIN -tags "novirt noaugeas"`

GOTAGS are transformed in our